src/screens/Login/index.tsx

```diff
@@ -0,0 +1,127 @@
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+import { AxiosError } from 'axios';
+
+import AuthAdapter from 'adapters/authAdapter';
+import logo from 'assets/images/logo.svg';
+import Button from 'components/Button';
+import Input from 'components/Input';
+import { setToken } from 'helpers/userToken';
+import { isEmailValid, isPasswordValid } from 'helpers/validators';
+
+import { PASSWORD_MIN_LENGTH } from '../../constants';
+
+function LoginScreen() {
+  const navigate = useNavigate();
+  const { t } = useTranslation('translation');
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [errors, setErrors] = useState<string[]>([]);
+  const [formSubmitted, setFormSubmitted] = useState(false);
+
+  const handleEmailChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setEmail(e.target.value);
+  };
+
+  const handlePasswordChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setPassword(e.target.value);
+  };
+
+  const performLogin = async () => {
+    try {
+      const response = await AuthAdapter.login({ email: email, password: password });
+
+      const {
+        attributes: { access_token: accessToken, refresh_token: refreshToken },
+      } = await response.data;
+
+      /* eslint-disable camelcase */
+      setToken({ access_token: accessToken, refresh_token: refreshToken });
+      /* eslint-enable camelcase */
+
+      navigate('/');
+    } catch (error) {
+      let errorMessage = t('login.generic-server-error');
+
+      if (error instanceof Error) {
+        errorMessage = (error as AxiosError).response?.data?.errors[0]?.detail || error.cause || errorMessage;
+      }
+      setErrors([errorMessage]);
+    } finally {
+      setFormSubmitted(false);
+    }
+  };
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    const formErrors = [];
+
+    if (!isEmailValid(email)) {
+      formErrors.push(t('login.invalid-email'));
+    }
+
+    if (!isPasswordValid(password)) {
+      formErrors.push(t('login.invalid-password', { passwordMinLength: PASSWORD_MIN_LENGTH }));
+    }
+
+    setErrors(formErrors);
+
+    if (formErrors.length === 0) {
+      setFormSubmitted(true);
+      performLogin();
+    }
+  };
+
+  return (
+    <>
+      <img className="inline-block" src={logo} alt="logo" />
+      <p data-test-id="login-header" className="my-8 text-white opacity-50">
+        {t('login.sign-in')} to Nimble
```

Comment:
I answered that in a previous PR - https://github.com/liamstevens111/react-survey-ic/pull/33#discussion_r1112694024

My thinking was that `Nimble` is the company name and I don't think we should "translate" the company name? I think it would be the same in all languages? Let me know if you think otherwise :)